rust-arroyo/src/backends/kafka/config.rs

```diff
@@ -11,17 +11,17 @@ pub struct OffsetResetConfig {
 
 #[derive(Debug, Clone)]
 pub struct KafkaConfig {
-    config_map: HashMap<String, String>,
+    config_map: FxHashMap<String, String>,
```

Comment:
This is only used once on startup, and probably not worth optimizing for.

Reply:
I agree, but there is no down-side of using `FxHashMap` compared to built-in HashMap.